src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining/InnerApi/Responses/GetEmployerFeedbackItem.cs

```diff
@@ -1,11 +1,51 @@
-﻿using System.Collections.Generic;
+﻿
+using System.Collections.Generic;
+using System.Linq;
+
 
 namespace SFA.DAS.FindApprenticeshipTraining.InnerApi.Responses
 {
     public class GetEmployerFeedbackItem
     {
         public IEnumerable<GetEmployerFeedbackAttributeItem> FeedbackAttributes { get; set; }
         public IEnumerable<GetEmployerFeedbackRatingItem> FeedbackRatings { get; set; }
+
+        public static implicit operator GetEmployerFeedbackItem(GetEmployerFeedbackResponse item)
+        {
+            if (item == null)
+                return null;
+
+            return new GetEmployerFeedbackItem()
+            {
+                FeedbackAttributes = item.ProviderAttribute.Select(a => new GetEmployerFeedbackAttributeItem()
+                {
+                    AttributeName = a.Name,
+                    Strength = a.Strength,
+                    Weakness = a.Weakness
+                }),
+                FeedbackRatings = new List<GetEmployerFeedbackRatingItem>()
+                {
+                    new GetEmployerFeedbackRatingItem() { FeedbackName = "Stars", FeedbackCount = item.Stars },
+                    new GetEmployerFeedbackRatingItem() { FeedbackName = "ReviewCount", FeedbackCount = item.ReviewCount },
+                }
+            };
+        }
+
+        public static implicit operator GetEmployerFeedbackItem(GetEmployerFeedbackSummaryItem item)
```

Comment:
As above - worth adding a couple of simple unit tests to prove happy and unhappy paths.

Reply:
done